src/style.js

```diff
@@ -21,15 +21,17 @@ export default class extends Component {
   }
 }
 
-export function flush() {
-  const ret = {}
-
-  for (const {props} of components) {
-    ret[props.styleId] = props.css
+function componentMap() {
+  const ret = new Map()
+  for (const c of components) {
+    ret.set(c.props.styleId, c)
```

Comment:
Can we make this backwards compatible? i.e. `ret.set(c.props.styleId, c.props.css)`?